source/UniverseObjects.cpp

```diff
@@ -294,15 +296,15 @@ void UniverseObjects::CheckReferences()
 			Logger::LogError("Warning: shipyard \"" + it.first + "\" is referred to, but has no ships.");
 	// System names are used by a number of classes.
 	for(auto &&it : systems)
-		if(it.second.Name().empty() && !NameIfDeferred(deferred["system"], it))
+		if(it.second.TrueName().empty() && !NameIfDeferred(deferred["system"], it))
 			NameAndWarn("system", it);
 	// Hazards are never serialized.
 	for(const auto &it : hazards)
 		if(!it.second.IsValid())
 			Warn("hazard", it.first);
 	// Wormholes are never serialized.
 	for(const auto &it : wormholes)
-		if(it.second.Name().empty())
+		if(it.second.DisplayName().empty())
```

Comment:
Shouldn't this be the true name?

Reply:
Wormholes have only a display name. When I was writing this, I was wondering if this warning even makes sense, because the name defaults to "???", but decided to left it as out of scope for this PR.